### PR TITLE
Use configuration path provided by ROSETTA_CONFIGURATION_FILE

### PR DESCRIPTION
**Pull request by sosedoff**
This change provides an option to specify the default config path with `ROSETTA_CONFIGURATION_FILE` environment variable if `--configuration-file` option is not provided. 

Improved use case looks like this:

```bash
export ROSETTA_CONFIGURATION_FILE=./path/to/my/config.json

$ rosetta-cli view:block XXX
$ rosetta-cli check:data
```

### CODE REVIEW

**Comment by patrick-ogrady** [comment]
❤️ ❤️

File: cmd/root.go [hunk 1]
```diff
@@ -214,6 +214,9 @@ default values.`,
 func initConfig() {
 	Context = context.Background()
 	var err error
+	if len(configurationFile) == 0 {
+		configurationFile = os.Getenv("ROSETTA_CONFIGURATION_FILE")
```
Comment (patrick-ogrady) [inline cmd/root.go:218]: @sosedoff, can you make this a constant? Would also appreciate a little comment blurb saying what we are doing (as the double `if len(configurationFile) == 0 {` could be confusing to others...maybe indicating that `os.Getenv` defaults to `""`)!

Comment (sosedoff) [reply cmd/root.go:218]: Yep, will do. I just wanted to see if you would be interested in the change in the first place. I will update the readme as well.

Comment (patrick-ogrady) [reply cmd/root.go:218]: 👍  Love the idea!

**Comment by patrick-ogrady** [comment]
You'll need to sign your commits before I can merge as well!

**Comment by sosedoff** [comment]
Updated per CR and signed the commits

**Review by patrick-ogrady — APPROVED** [review]
Awesome UX improvement!

**Pull request merged**